code/modules/surgery/surgery.dm

```diff
@@ -508,13 +508,13 @@
 
 	var/germs = 0
 
-	for(var/mob/living/carbon/human/H in view(2, E.loc))//germs from people
-		if(length(get_path_to(E.loc, H.loc, max_distance = 2, simulated_only = FALSE)))
+	for(var/mob/living/carbon/human/H in view(2, E.owner))//germs from people
```

Comment:
Do you think you could modify this to check for owner if it exists, and otherwise just use loc? In the off-chance this gets accidentally used in a surgery or something after an organ gets removed.

Reply:
Returns if no owner now since it really shouldn't be used on organs outside people